src/Features/CSharp/Portable/UseLocalFunction/CSharpUseLocalFunctionCodeFixProvider.cs

```diff
@@ -56,8 +58,17 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
 
                 nodesToTrack.Add(localDeclaration);
                 nodesToTrack.Add(lambda);
+
+                if (diagnostic.AdditionalLocations.Count > 2)
+                {
+                    for (var i = 2; i < diagnostic.AdditionalLocations.Count; i++)
+                    {
+                        memberAccess.Add((MemberAccessExpressionSyntax)diagnostic.AdditionalLocations[i].FindNode(cancellationToken));
```

Comment:
you need to pass getInnermostNodeForTie: true.

Otherwise, this will crash if yo uhave something like:    Foo(localFunc.Invoke())

(please add a test for this case).